apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -837,25 +891,43 @@ def test(element_html: str, data: pl.ElementTestData) -> None:
     # TODO: The only wrong answer being tested is the correct answer with the first
     # block mising. We should instead do a random selection of correct and incorrect blocks.
     elif data["test_type"] == "incorrect":
-        answer = filter_multiple_from_array(
-            data["correct_answers"][answer_name], ["inner_html", "indent", "uuid"]
-        )
+        answer = deepcopy(data["correct_answers"][answer_name])
+        distractor_feedback = {
+            item["uuid"]: item["distractor_feedback"]
+            for item in data["params"][answer_name]
+            if not item["is_correct"]
+        }
+
         answer.pop(0)
         score = 0
         if grading_method is GradingMethodType.UNORDERED or (
             (
-                grading_method is GradingMethodType.DAG
-                or grading_method is GradingMethodType.RANKING
+                grading_method in [GradingMethodType.DAG, GradingMethodType.RANKING]
```

Comment:
Similar to the feedback type list, this list can hardcoded once (it's constructed twice in this file).

Reply:
Its actually more places than two. I don't really want to change this right now though because its almost completely unrelated to the actual content of this PR. I also don't think there is an obvious thing to call this list as there was with the feedback list.